README.md

```diff
@@ -42,7 +42,8 @@ Unit testing is done via jest.
 
 We use [cypress](https://cypress.io) for e2e tests.
 
-1. Start the frontend with `yarn start`
+1. Start the frontend with `yarn start:test` in dev test mode or build a test build with `yarn build:test` which you can serve with `yarn build:serve`
```

Comment:
What about renaming the serve task? With our new schema this looks as whether it builds the app in a "serve"-variant. Therefore I'd suggest to simply swap the terms build and serve.

Reply:
good point. done